Lib/test/test_ssl.py

```diff
@@ -4720,6 +4720,28 @@ def test_internal_chain_client(self):
                     ssl.PEM_cert_to_DER_cert(pem), der
                 )
 
+    def test_certificate_chain(self):
+        client_context, server_context, hostname = testing_context(
+            server_chain=False
+        )
+        server = ThreadedEchoServer(context=server_context, chatty=False)
+        with server:
+            with client_context.wrap_socket(
+                socket.socket(),
+                server_hostname=hostname
+            ) as s:
+                s.connect((HOST, server.port))
+                vc = s.get_verified_chain()
+                self.assertEqual(len(vc), 2)
+
+                ee, ca = vc
```

Comment:
I would add asserts that `ee` and `ca` are both of type `bytes`. Otherwise the test would also pass without the changes in Lib/ssl.py.

Reply:
Proposed improvement applied.